internal/server/server.go

```diff
@@ -259,6 +331,26 @@ func (h *Handlers) ComposeImage(ctx echo.Context) error {
 	if err != nil {
 		return err
 	}
+
+	rawCR, err := json.Marshal(composeRequest)
+	if err != nil {
+		return err
+	}
+
+	ih := ctx.Get("IdentityHeader")
+	if ih == nil {
+		return echo.NewHTTPError(http.StatusInternalServerError, "Identity Header missing in request handler")
+	}
+	idHeader, ok := ih.(IdentityHeader)
+	if !ok {
+		return echo.NewHTTPError(http.StatusInternalServerError, "Identity Header invalid in request handler")
+	}
+
+	err = h.server.db.InsertCompose(composeResult.Id, idHeader.Identity.AccountNumber, idHeader.Identity.Internal.OrgId, rawCR)
```

Comment:
In the future we may want to do this before creating the actual compose request, to make sure we only schedule a compose once we have successfully recorded it. Especially if we enforce caps.

Reply:
I agree, but the problem is we need the job uuid as that's the primary key here. What we could do is of course insert it with a random key, and then once we get the actual uuid, update the row.

Reply:
Ah.